apis/v1alpha1/gateway_types.go

```diff
@@ -134,7 +134,7 @@ type GatewaySpec struct {
 // validating webhook.
 type Listener struct {
 	// Hostname specifies the virtual hostname to match for protocol types that
-	// define this concept. When unspecified, "", or "*", all hostnames are
+	// define this concept. When unspecified, "", or "&ast;", all hostnames are
```

Comment:
This change feels strange, is it required for the reference docs?

Reply:
In markdown the `*` triggers italicizing, so it never appears in the rendered API reference at all. This is the only workaround I could find. Happy to concede that it sucks :)

Reply:
Oh yep, that makes a lot of sense. Any chance we can update the generator to escape potential formatting like this? This looks awfully weird in a Go file and feels like something that could be misconstrued by someone referring to go types as reference.

Reply:
fixed.

Reply:
fixed-ish